extensions/amp-3q-player/0.1/amp-3q-player.js

```diff
@@ -200,6 +204,19 @@ class Amp3QPlayer extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Forces the Layout to 'container' and sets a new height
+   * @private
+   * @param {number} newHeight
+   */
+  resize_(newHeight) {
+    this.attemptChangeHeight(newHeight).then(() => {
+      this.notifyChangedHeight_();
```

Comment:
When I suggested `notifyChangeHeight/Rejected_` I meant that you should implement them to notify the iframe in case the render state goes out of sync. If the iframe doesn't fail if the resize request is rejected, it's not necessary.